x-pack/plugins/lens/public/datasources/form_based/operations/definitions/calculations/counter_rate.tsx

```diff
@@ -75,7 +75,16 @@ export const counterRateOperation: OperationDefinition<
       column.timeShift
     );
   },
-  toExpression: (layer, columnId) => {
+  toExpression: (layer, columnId, indexPattern) => {
```

Comment:
if we detect configuration which can be optimized we'll just run identity function on the client (this could be further optimized by making sure correct column id is provided to the max agg)